Dockerfile_Production

```diff
@@ -0,0 +1,23 @@
+FROM node:alpine as build
+
+WORKDIR /app
+
+COPY package.json .
+
+RUN npm install
+
+COPY . .
+
+RUN npm run build
+
+FROM nginx
+
+COPY --from=build /app/build /usr/share/nginx/html/frontend-service/exam-attendance
+
+RUN rm -f /etc/nginx/conf.d/default.conf
+
+COPY  nginx.conf  /etc/nginx/conf.d/default.conf
+
+EXPOSE 80
+
```

Comment:
Group the related commands since docker runs by stages when we add extra lines it will display as separate stages for each line.

Reply:
Pushed the changes